WordPressAuthenticator/Sources/NUX/Button/NUXButtonViewController.swift

```diff
@@ -1,10 +1,16 @@
 import UIKit
 import WordPressKit
+import WordPressShared
 
-@objc public protocol NUXButtonViewControllerDelegate {
+public protocol NUXButtonViewControllerDelegate: AnyObject {
     func primaryButtonPressed()
-    @objc optional func secondaryButtonPressed()
-    @objc optional func tertiaryButtonPressed()
+    func secondaryButtonPressed()
+    func tertiaryButtonPressed()
```

Comment:
It's a safe change because it never used dynamically from Objective-C (think `respondsToSelector:`  to dynamically configure the UI.